\section{Supplemental Methods}


\textbf{Soil Collection and Preparation}



Soils were collected from an organic farm in Penn Yan, New York.  These soils are characterized as Honoeye/Lima, a silty clay loam on calcareous bedrock. To get a field average, 10cm cores were collected (in duplicate) from six different sampling locations around the field using a slide hammer bulk density sampler (coordinates: (1) N 42° 40.288’ W 77° 02.438’, (2) N 42° 40.296’ W 77° 02.438’, (3) N 42° 40.309’ W 77° 02.445’, (4) N 42° 40.333’ W 77° 02.425’, (5) N 42° 40.340’ W 77° 02.420’, (6) N 42° 40.353’ W 77° 02.417’) on November 21, 2011.   Cores were all sieved through a 2mm sieve, homogenized by mixing, and stored at 4°C until setup in microcosms for pretreatment (within 1-2 week of collection).  Carbon and nitrogen content , as well as, water holding capacity were previously measured for these soils \cite{Berthrong_2013}. Reported values for the organic field were: 12.15 (+/- 0.78) mg C g\textsuperscript{-1} dry soil, 1.16 (+/- 0.13) mg N g\textsuperscript{-1} dry soil,



\textbf{Cellulose production}



Bacterial cellulose was produced by \textit{Gluconoacetobacter xylinus} grown in Heo and Son \cite{Heo_2002} liquid minimal media made with 0.1\% glucose (one batch with \textsuperscript{12}C- and another with \textsuperscript{13}C-glucose). All cellulose (\textsuperscript{12}C and \textsuperscript{13}C) were produced in 1L Erlenmeyer flask containing 100 mL Heo and Son minimal media that were inoculated with three isolated colonies of \textit{Gluconoacetobacter xylinus} grown on Heo and Son 0.1\% glucose agar plates (using \textsuperscript{12}C-glucose) at 30°C without inositol. Flasks were incubated statically in the dark at 30°C for 2-3 weeks until thick cellulose pellicule had formed.  Cellulose pellicules were collected and washed with two parts 1\% alconox was added to each flask and autoclaved. Cellulose pellicules were purified by repeated (10x) overnight dialysis in 1 L deionized water. Harvested pellicules were dried overnight (60°C) and then cut into pieces and ground using ball grinder until desired size range (53$\mu$m - 250$\mu$m) was achieved (checked by drying sieving). Size range was based on particulate organic matter to emulate how microbes may experience cellulose in the environment \cite{Cambardella_1992} and for even distribution in microcosms. 

Post processing, purity of ground cellulose was checked with \textit{E.coli} cultures, Benedict's reducing sugars assay, Bradford assay, and isotopic analysis. \textit{E.coli} is not able to use cellulose as a C source but is capable of growth on a variety of nutrients available in the Heo and Son medium.  Biological assays consisted of E. coli inoculated into minimal M9 media which lacked a carbon source and was supplemented with either: (1) 0.01\% glucose, (2) 2.5 mg purified, ground cellulose, (3) 25 mg purified, ground cellulose, (4) 25 mg purified, ground cellulose and 0.01\% glucose. Growth in media was checked by spectrometer (OD450). No measureable growth was observed with either 2 mg or 25 mg cellulose, indicating absence of contaminating nutrients. In addition, the presence of 25 mg cellulose did not inhibit the growth of \textit{E.coli} cultures provided with glucose relative to control, indicating the absence of compounds that may inhibit microbial growth in the purified cellulose. 

Purified cellulose was also assayed for residual proteins and sugars using Bradford and Benedict's assays, respectively. Bradford assay was performed as in Bradford \cite{Bradford_1976} with a standard curve ranging from 0-2000$\mu$gml \textsuperscript{-1} BSA. Ground, purified cellulose contained 6.92 $\mu$g protein mg cellulose\textsuperscript{-1} (ie. 99.31\% purity). Reducing sugars were not detected in cellulose using Benedict's reducing sugar assay \cite{benedict1909reagent} tested at 10mg cellulose ml \textsuperscipt{-1}. Finally, \textsuperscript{13}C-cellulose had an average 96\%$\pm$5 (s.d.) degree of \textsuperscript{13}C labeling as determined by isotopic analysis (UCDavis Stable Isotope Facility).           

\textbf{Soil microcosms}

A subset of soil was dried at 105°C overnight to determine soil moisture content gravimetrically. Microcosms (35 total) were created by adding the equivalent of ten grams approximate dry soil weight of the sieved soil to a 250mL Erlenmeyer flask capped with a butyl rubber stopper to preventing drying. Microcosms were preincubated at 25°C for 2 weeks until the soil respiration rate (determined by GCMS measurement of head space CO\textsubscript{2}) had stabilized. Sieving causes a transient increase in soil respiration rate presumably due to the liberation of fresh labile soil organic matter (ref). Pre-incubation ensures that this labile organic matter is consumed and/or stabilized prior to the beginning of the experiment. Respiration rate (CO\textsubscript{2}) began to plateau around 10 days, with no change in rate after that time. Stoppers were removed for 10 min every 3 days to exchange the headspace with air. 

Each microcosm received an evenly distributed dry addition of insoluble substrates (2 mg cellulose and 1.2 mg lignin g dry soil\textsuperscript{-1}) and a liquid addition (1.2 mL) of a complex substrate mixture. The complete amendment (dry and liquid additions) was added to each microcosm at 5.3 mg g dry soil\textsuperscript{-1}; representative of natural concentrations \cite{Schneckenberger_2008}. The complex mixture was designed based on switch grass biomass composition \cite{Yan_2010,David_2010} to include (by mass) 38\% cellulose, 23\% lignin, 20\% xylose, 3\% arabinose, 1\% galactose, 1\% glucose, and 0.5\% mannose, with the remaining 13.5\% mass composed of amino acids (in-house made replica of teknova Cat#C0705) and basal salt mixture (Murashige and Skoog, Sigma M5524). The volume of the liquid addition was chosen to achieve 50\% water holding capacity of the soil. Water holding capacity of 50\% was chosen to achieve $\sim$70\% water filled pore space in these soils based on soil texture, which is the optimal water content for respiration \cite{Linn_1984, Linn_1984}.

Three parallel treatments were performed; (1) control, all \textsuperscript{12}C-carbons, (2) cellulose, all \textsuperscript{12}C-carbons except \textsuperscript{13}C-cellulose, (3) xylose, all \textsuperscript{12}C-carbons except \textsuperscript{13}C-xylose.  Replicate microcosms were harvested (stored at -80°C until nucleic acid processing) at days 1 (control and xylose only), 3, 7, 14, and 30. A subset of microcosm soil for each treatment and time point were isotopically analyzed at Cornell University Stable Isotope Laboratory to determine amount of \textsuperscript{13}C that remained at each time point.                       



\textbf{Nucleic acid extraction}

DNA was extracted from 0.25g soil using a modified Griffiths procotol \cite{Griffiths_2000}.  Soils were bead beat in 2mL lysis tubes containing 0.5g silica/zirconia beads (treated at 300°C for 4 hours to remove RNases), 0.5mL extraction buffer (240 mM Phosphate buffer
0.5\% N-lauryl sarcosine), and 0.5mL phenol-chloroform-isoamyl alcohol (25:24:1) for 1 min at 5.5 ms\textsuperscript{-1}. After beat beading, 85uL 5M NaCl and 60uL 10\% hexadecyltriammonium bromide (CTAB)/0.7M NaCl were added to lysis tube, vortexed, chilled for 1min on ice, and centrifuged at 16,000 x g for 5min at 4°C.  The aqueous layer was transferred to a new tube and reserved on ice.  To increase DNA recovery, a double extraction was performed by adding 85uL 5M NaCl and 0.5mL extraction buffer to same lysis tube, vortexed, and centrifuged same as before.  New aqueous layer was added to previously collected aqueous layer and washed with 0.5mL chloroform:isoamyl alcohol (24:1), vortexed, and centrifuged same as before.  Aqueous layer was transferred to a new tube and nucleic acids were precipitated using 2 volumes polyethylene glycol solution (30\% PEG 8000, 1.6M NaCl) on ice for 2hrs, followed by centrifugation at 16,000 x g, 4°C for 30min.  Supernatant was discarded and pellets were washed with 1mL ice cold 70\% EtOH; each wash consisting of a vortex followed by centrifugation at 16,000 x g, 4°C for 10min.  Pellets were air dried, resuspended in 50uL TE, replicate extractions pooled, and stored at -20°C. To prepare nucleic acid extracts for isopycnic centrifugation as presviously described \cite{Buckley_2007}, DNA was size selected (>4kb) using 1\% low melt agarose gel and $\beta$- agarase I enzyme extraction per manufacturers protocol (New England Biolab, M0392S).  Final resuspension of DNA pellet was in 50$\mu$L TE.   


\textbf{Isopycnic centrifugation and fractionation}

 Isopycnic gradients were setup using a modified protocol \cite{Neufeld_2007} for five \textsuperscript{12}C-control, five \textsuperscript{13}C-xylose, and four \textsuperscript{13}C-cellulose microcosms; one per each time point. A density gradient solution of 1.762 g cesium chloride (CsCl) ml\textsuperscript{-1} in gradient buffer solution (pH 8.0 15mM Tris-HCl, 15mM EDTA, 15mM KCl) was used to separate \textsuperscript{13}C-enriched and \textsuperscript{12}C-nonenriched DNA. Each gradient was loaded with approximately 5$\mu$g of DNA into a Beckman OptiSeal\textsuperscript{TM} polyallomer centrifuge tube (part no. 361621; Beckman, Palo Alto, CA) containing the cesium chloride-gradient buffer solution. Samples were centrifuged on a Beckman Coulter Optima\textsuperscript{TM} MAX-E ultracentrifuge using a TLA-110 fixed-angle rotor for 66 h at 55,000 rpm and room temperature (RT). Fractions of $\sim$100 $\mu$L were collected into Acroprep\textsuperscript{TM} 96 filter plate (part no. 5035, Pall Life Sciences) by displacing the DNA-CsCl-gradient buffer solution in the centrifugation tube with water. The refractive index of each fraction was measured using a Reichart AR200 digital refractometer. Then buoyant density was calculated from the refractive index using the equation $\rho$=\textit{a}$\eta$-\textit{b}, where $\rho$ is the density of the CsCl (g ml\textsuperscript{-1}), $\eta$ is the measured refractive index, and \textit{a} and \textit{b} are coefficient values of 10.9276 and 13.593, respectively, for CsCl at 20°C \cite{9780408708036}. The collected DNA fractions were purified by washing each with 200$\mu$L TE, centrifuging for 10min at 500 x g and discarding flow through. This was repeated five more times except using 300$\mu$L of TE in subsequent washes and centrifuging for 20min on the final wash. Finally, 50$\mu$L TE was added to each fraction, incubated on filter for 5min at RT then resuspended DNA was pipetted off the filter into a new microfuge tube.   Fractions were quantitated by qPCR (BioRad) using 12.5$\mu$L QuantiFast  SYBR greenPCR  master mix (204056, Qiagen, Valencia, CA), 1.25$\mu$L 10$\mu$M 515F (5'-GTGCCAGCMGCCGCGGTAA -3'), 1.25$\mu$L 10$\mu$M 806R (5'-GGACTACHVGGGTWTCTAAT-3'), and 1:100 dilution of template (include PCR cycles? or are they previously published?). For every gradient, 20 fractions were chosen for sequencing between the density range 1.67-1.75g mL\textsuperscript{-1}.      


\textbf{Sequencing}  Barcoded 454 primers were designed using 454-specific adapter B, 10bp barcodes \cite{Hamady_2008}, a 2bp linker (put those bp in here), and 806R primer for reverse primer (BA806R); and 454-specific adapter A, a 2bp linker, and 515F primer for forward primer (BA515F). Each fraction was PCR amplified using 0.25$\mu$L 5U/$\mu$l AmpliTaq Gold (N8080243, Life Technologies, Grand Island, NY), 2.5$\mu$L 10X Buffer II (100 mM Tris-HCl, pH 8.3, 500 mM KCl), 2.5$\mu$L 25mM MgCl_{2}, 4$\mu$L 5mM dNTP, 1.25$\mu$L 10mg/mL BSA, 0.5$\mu$L 10$\mu$M BA515F, 1$\mu$L 5$\mu$M BA806R,3$\mu$L H_{2}O, 10$\mu$L 1:30 DNA template) in triplicate and checked by 1\% agarose gel. Replicate reactions were then pooled, normalized either using Pico green quantification and manual calculation or by SequalPrep\textsuperscript{TM} normalization plates (A10510, Invitrogen, Carlsbad, CA), then pooled in equimolar concentrations.  Overall pool was gel extracted from a 1\% agarose gel using Wizard SV gel and PCR clean-up system (A9281, Promega, Madison, WI) per manufacturer's protocol.  Amplicons were sequenced on Roche 454 FLX system using titanium chemistry at Selah Genomics (formerly EnGenCore, Columbia, SC)    

\textbf{Post-Sequencing Analysis}
 