\section{Supplemental Methods}


\textbf{Soil Collection and Preparation}



Soils were collected from an organic farm in Penn Yan, New York.  These soils are characterized as Honoeye/Lima, a silty clay loam on calcareous bedrock. To get a field average, cores (5 cm diameter x 10 cm depth) were collected in duplicate from six different randomized sampling locations around the field using a slide hammer bulk density sampler (coordinates: (1) N 42° 40.288’ W 77° 02.438’, (2) N 42° 40.296’ W 77° 02.438’, (3) N 42° 40.309’ W 77° 02.445’, (4) N 42° 40.333’ W 77° 02.425’, (5) N 42° 40.340’ W 77° 02.420’, (6) N 42° 40.353’ W 77° 02.417’) on November 21, 2011. Cores were all sieved through a 2mm sieve, homogenized by mixing, and stored at 4°C until setup for preincubation (within 1-2 week of collection).  Carbon and nitrogen content were previously measured for these soils \cite{Berthrong_2013}. Reported values for the organic field were 12.15 ($\pm$ s.d. 0.78) mg C g\textsuperscript{-1} dry soil and 1.16 ($\pm$ s.d. 0.13) mg N g\textsuperscript{-1} dry soil.



\textbf{Cellulose production}

Bacterial cellulose was produced by \textit{Gluconoacetobacter xylinus} grown in Heo and Son \cite{Heo_2002} liquid minimal media made with 0.1\% glucose (one batch with \textsuperscript{12}C- and another with \textsuperscript{13}C-glucose). All cellulose (\textsuperscript{12}C and \textsuperscript{13}C) were produced in 1L Erlenmeyer flask containing 100 mL Heo and Son minimal media that were inoculated with three isolated colonies of \textit{Gluconoacetobacter xylinus} grown on Heo and Son 0.1\% glucose agar plates (using \textsuperscript{12}C-glucose) at 30°C without inositol. Flasks were incubated statically in the dark at 30°C for 2-3 weeks until thick cellulose pellicule had formed.  Cellulose pellicules were collected and washed with two parts 1\% alconox and autoclaved. Cellulose pellicules were purified by repeated (10x) overnight dialysis in 1 L deionized water. Harvested pellicules were dried overnight (60°C) and then cut into pieces and ground using ball grinder until desired size range (53$\mu$m - 250$\mu$m) was achieved (checked by drying sieving). Size range was based on particulate organic matter to emulate how microbes may experience cellulose in the environment \cite{Cambardella_1992} and for even distribution in microcosms. 

Post processing, purity of ground cellulose was checked with \textit{E.coli} cultures, Benedict's reducing sugars assay, Bradford assay, and isotopic analysis. \textit{E.coli} is not able to use cellulose as a C source but is capable of growth on a variety of nutrients available in the Heo and Son medium.  Biological assays consisted of E. coli inoculated into minimal M9 media which lacked a carbon source and was supplemented with either: (1) 0.01\% glucose, (2) 2.5 mg purified, ground cellulose, (3) 25 mg purified, ground cellulose, (4) 25 mg purified, ground cellulose and 0.01\% glucose. Growth in media was checked by spectrometer (OD450). No measureable growth was observed with either 2 mg or 25 mg cellulose, indicating absence of contaminating nutrients. In addition, the presence of 25 mg cellulose did not inhibit the growth of \textit{E.coli} cultures provided with glucose relative to control, indicating the absence of compounds that may inhibit microbial growth in the purified cellulose. 

Purified cellulose was also assayed for residual proteins and sugars using Bradford and Benedict's assays, respectively. Bradford assay was performed as in Bradford \cite{Bradford_1976} with a standard curve ranging from 0 - 2000 $\mu$g ml \textsuperscript{-1} BSA. Ground, purified cellulose contained 6.92 $\mu$g protein mg cellulose\textsuperscript{-1} (\textit{i.e.} 99.31\% purity). Reducing sugars were not detected in cellulose using Benedict's reducing sugar assay \cite{benedict1909reagent} tested at 10 mg cellulose ml\textsuperscript{-1}. Finally, \textsuperscript{13}C-cellulose had an average 96\% $\pm$ 5 (s.d.) degree of \textsuperscript{13}C labeling as determined by isotopic analysis (UCDavis Stable Isotope Facility).           

\textbf{Soil microcosms}

A subset of soil was dried at 105°C overnight to determine soil moisture content gravimetrically. Microcosms (35 total) were created by adding the equivalent of 10 g approximate dry soil weight of the sieved soil to a 250 mL Erlenmeyer flask capped with a butyl rubber stopper to prevent drying. Microcosms were preincubated at 25°C for 2 weeks until the soil respiration rate (determined by GCMS measurement of head space CO$_{2}$) had stabilized. Sieving causes a transient increase in soil respiration rate presumably due to the liberation of fresh labile soil organic matter \cite{Datta_2014}. Pre-incubation ensures that this labile organic matter is consumed and/or stabilized prior to the beginning of the experiment. Respiration rate (CO$_{2}$) began to plateau around 10 days, with no change in rate after that time. Stoppers were removed for 10 min every 3 days to exchange the headspace with air. 

Three parallel treatments were performed with identical amendments of carbon which varied only with respect to \textsuperscript{13}C-labelling as follows: (1) unlabeled control,(2)\textsuperscript{13}C-cellulose (synthesis and purity described above), (3)\textsuperscript{13}C-xylose (98 atom\% \textsuperscript{13}C, Sigma Aldrich 666378). Each treatment had 2 replicates per time point (n = 4) except day 30 which had 4 replicates; total microcosms per treatment n = 12, except \textsuperscript{13}C-cellulose which was not sampled at day 1, n = 10. Each microcosm received an evenly distributed dry addition of insoluble substrates (2 mg cellulose and 1.2 mg lignin g dry soil\textsuperscript{-1}) and a liquid addition (1.2 mL) of a complex substrate mixture. The complete amendment (dry and liquid additions) was added to each microcosm at 5.3 mg g dry soil\textsuperscript{-1}; representative of natural concentrations \cite{Schneckenberger_2008}. The complex mixture was designed based on switch grass biomass composition \cite{Yan_2010,David_2010} to include (by mass) 38\% cellulose, 23\% lignin, 20\% xylose, 3\% arabinose, 1\% galactose, 1\% glucose, and 0.5\% mannose, with the remaining 13.5\% mass composed of amino acids (in-house made replica of teknova Cat#C0705) and basal salt mixture (Murashige and Skoog, Sigma M5524) for a final C:N of 10. The volume of the liquid addition was chosen to achieve 50\% water holding capacity of the soil. Water holding capacity of 50\% was chosen to achieve $\sim$70\% water filled pore space in these soils based on soil texture, which is the optimal water content for respiration \cite{Linn_1984,Linn_1984}.

Replicate microcosms were harvested (stored at -80°C until nucleic acid processing) at days 1 (control and xylose only), 3, 7, 14, and 30. A subset of microcosm soil for each treatment and time point were isotopically analyzed at Cornell University Stable Isotope Laboratory to determine amount of \textsuperscript{13}C that remained at each time point.   

\textbf{Nucleic acid extraction}

Nucleic acids were extracted from 0.25 g soil using a modified Griffiths procotol \cite{Griffiths_2000}. Cell lysis was performed by bead beating for 1 min at 5.5 ms\textsuperscript{-1} in 2mL lysis tubes containing 0.5 g of 0.1 mm diameter silica/zirconia beads (treated at 300°C for 4 hours to remove RNases), 0.5 mL extraction buffer (240 mM Phosphate buffer 0.5\% N-lauryl sarcosine), and 0.5 mL phenol-chloroform-isoamyl alcohol (25:24:1) for 1 min at 5.5 ms\textsuperscript{-1}. After lysis, 85 uL 5 M NaCl and 60 uL 10\% hexadecyltriammonium bromide (CTAB)/0.7 M NaCl were added to lysis tube, vortexed, chilled for 1 min on ice, and centrifuged at 16,000 x g for 5 min at 4°C. The aqueous layer was transferred to a new tube and reserved on ice. To increase DNA recovery, the pellet was back extracted with 85 uL 5 M NaCl and 0.5 mL extraction buffer. The aqueous extract was washed with 0.5 mL chloroform:isoamyl alcohol (24:1). Nucleic acids were precipitated by addition of 2 volumes polyethylene glycol solution (30\% PEG 8000, 1.6 M NaCl) on ice for 2 hrs, followed by centrifugation at 16,000 x g, 4°C for 30 min. The supernatant was discarded and pellets were washed with 1 mL ice cold 70\% EtOH. Pellets were air dried, resuspended in 50 uL TE and stored at -20°C. To prepare nucleic acid extracts for isopycnic centrifugation as presviously described \cite{Buckley_2007}, DNA was size selected (\textgreater 4kb) using 1\% low melt agarose gel and $\beta$-agarase I enzyme extraction per manufacturers protocol (New England Biolab, M0392S).  Final resuspension of DNA pellet was in 50 $\mu$L TE.   


\textbf{Isopycnic centrifugation and fractionation}

For each time point in the series isopycnic gradients were setup using a modified protocol \cite{Neufeld_2007} for a total of five \textsuperscript{12}C-control, five \textsuperscript{13}C-xylose, and four \textsuperscript{13}C-cellulose microcosms. A density gradient (average density 1.69 g mL\textsuperscript{-1}) solution of 1.762 g cesium chloride (CsCl) ml\textsuperscript{-1} in gradient buffer solution (pH 8.0 15 mM Tris-HCl, 15 mM EDTA, 15 mM KCl) was used to separate \textsuperscript{13}C-enriched and \textsuperscript{12}C-nonenriched DNA. Each gradient was loaded with approximately 5 $\mu$g of DNA and centrifuged on a Beckman Coulter Optima\textsuperscript{TM} MAX-E ultracentrifuge using a TLA-110 fixed-angle rotor for 66 h at 55,000 rpm and room temperature (RT). Fractions of $\sim$100 $\mu$L were collected from below by displacing the DNA-CsCl-gradient buffer solution in the centrifugation tube with water using a syringe pump at a flow rate of 3.3 $\mu$L s\textsuperscript{-1} \cite{Manefield_2002} into Acroprep\textsuperscript{TM} 96 filter plate (part no. 5035, Pall Life Sciences). The refractive index of each fraction was measured using a Reichart AR200 digital refractometer modified as previously described \cite{Buckley_2007} to measure a volume of 5 $\mu$L. Then buoyant density was calculated from the refractive index as previously described \cite{Buckley_2007} using the equation $\rho$=\textit{a}$\eta$-\textit{b}, where $\rho$ is the density of the CsCl (g ml\textsuperscript{-1}), $\eta$ is the measured refractive index, and \textit{a} and \textit{b} are coefficient values of 10.9276 and 13.593, respectively, for CsCl at 20°C \cite{9780408708036}. The collected DNA fractions were purified by repetitive washing of Acroprep filter wells with TE. Finally, 50 $\mu$L TE was added to each fraction then resuspended DNA was pipetted off the filter into a new microfuge tube. The number of 16S rRNA genes of each fraction were quantitated by qPCR (Bio-Rad C1000/CFX96 thermocycler) as described previously \cite{Berthrong_2013} using 12.5 $\mu$L QuantiFast SYBR green PCR  master mix (Qiagen 204056), 1.25 $\mu$L 10 $\mu$M 515F (5'-GTGCCAGCMGCCGCGGTAA -3'), 1.25 $\mu$L 10 $\mu$M 806R (5'-GGACTACHVGGGTWTCTAAT-3'), and 1:100 dilution of DNA template. To estimate the abundances of rRNA gene copies, we used standard curves from 10-fold serial dilutions of 16S generated from \textit{Klebsiella pneumonia} using the same primers.   

\textbf{DNA Sequencing}  For every gradient, 20 fractions were chosen for sequencing between the density range 1.67-1.75 g mL\textsuperscript{-1}. A total of 14 gradients (280 fractions) and their corresponding bulk DNA extraction (after $\beta$-agarase size selection) were amplified for sequencing. Barcoded 454 primers were designed using 454-specific adapter B, 10 bp barcodes \cite{Hamady_2008}, a 2 bp linker (5'-CA-3'), and 806R primer for reverse primer (BA806R); and 454-specific adapter A, a 2 bp linker (5'-TC-3'), and 515F primer for forward primer (BA515F). Each fraction was PCR amplified using 0.25 $\mu$L 5U $\mu$l\textsuperscript{-1} AmpliTaq Gold (Life Technologies, Grand Island, NY; N8080243), 2.5 $\mu$L 10X Buffer II (100 mM Tris-HCl, pH 8.3, 500 mM KCl), 2.5 $\mu$L 25 mM MgCl$_{2}$, 4 $\mu$L 5 mM dNTP, 1.25 $\mu$L 10 mg mL\textsuperscript{-1} BSA, 0.5 $\mu$L 10 $\mu$M BA515F, 1 $\mu$L 5 $\mu$M BA806R, 3 $\mu$L H$_{2}$O, 10 $\mu$L 1:30 DNA template) in triplicate and checked by 1\% agarose gel. Samples were normalized either using Pico green quantification and manual calculation or by SequalPrep\textsuperscript{TM} normalization plates (Invitrogen, Carlsbad, CA; A10510), then pooled in equimolar concentrations. Pooled DNA was gel extracted from a 1\% agarose gel using Wizard SV gel and PCR clean-up system (Promega, Madison, WI; A9281) per manufacturer's protocol.  Amplicons were sequenced on Roche 454 FLX system using titanium chemistry at Selah Genomics (formerly EnGenCore, Columbia, SC)    

\textbf{Post-Sequencing Analysis}
\subsubsection{Sequence quality control}
Sequences were initially screened by maximum expected errors at a specific read
length threshold \citep{Edgar_2013} which has been shown to be as effective as
denoising with respect to removing pyrosequencing errors. Specifically, reads
were first truncated to 250 nucleotides (nt) (all reads shorter than 250 nt
were discarded) and any read that exceeded a maximum expected error threshold
of 0.5 was removed. After truncation and max expected error trimming, 87\% of
original reads remained. Forward primer and barcode was then removed from the
high quality, truncated reads.  Remaining reads were taxonomically annotated
using the "UClust" taxonomic annotation framework in the QIIME software package
\citep{Edgar_2010,Caporaso_2010} with cluster seeds from Silva SSU rRNA database
\citep{Pruesse_2007} 97\% sequence identity OTUs as reference (release 111Ref).
Reads annotated as "Chloroplast", "Eukaryota", "Archaea", "Unassigned" or
"mitochondria" were culled from the dataset. Finally, reads were aligned to the
Silva reference alignment provided by the Mothur software package
\cite{Schloss_2009} using the Mothur NAST aligner \cite{DeSantis_2006}. All reads that
did not align to the expected amplicon region of the SSU rRNA gene were
discarded. Quality control parameters removed 344,472 of 1,720,480 raw reads.

\subsubsection{Sequence clustering}
Sequences were distributed into OTUs using the UParse methodology
\citep{Edgar_2013}. Specifically, OTU centroids (i.e. seeds) were identified
using USearch on non-redundant reads sorted by count. The sequence
identity threshold for establishing a new OTU centroid was 97\%. 
With USearch/UParse,
potential chimeras are identified during OTU centroid selection and are not
allowed to become cluster centroids effectively removing chimeras from the read
pool. All quality controlled reads were then mapped to cluster centroids at an
identity threshold of 97\% again using USearch. 97\% of quality
controlled reads could be mapped to centroids. Unmapped reads do not count
towards sample counts and are removed from downstream analyses. The
USearch software version for cluster generation was 7.0.1090.

\subsubsection{Phylogenetic analysis}
Alignment of OTU centroid SSU rRNA genes was done with SSU-Align which is based on Infernal
\citep{Nawrocki_2013, Nawrocki_2009}. Columns in the alignment that were not included in
the SSU-Align covariance models or were aligned with poor confidence (less than
95\% of characters in a position had posterior probability alignment scores of
at least 95\%) were masked for phylogenetic reconstruction. Additionally, the
alignment was trimmed to coordinates such that all sequences in the alignment
began and ended at the same positions. FastTree \cite{Price_2009} was used to
reconstruct the phylogeny.

\subsubsection{Identifying OTUs that incorporated $^{13}$C into their DNA}
DNA-SIP is a culture-independent approach towards defining identity-function
connections in microbial communities \cite{Buckley_2011, Neufeld_2007, Radajewski_2003}. . Microbes are identified on the basis of
isotope assimilation into DNA. As the bouyant density (BD) of a macromolecule is dependent on many factors in addition to stable isotope incorporation (e.g. GC-content in nucleic acids \cite{Youngblut_2014}),
labeled nucleic acids from one microbial population may have the same BD as unlabeled nucleic acids from another. Therefore, it is imperative to compare results of isotopic labelling to results obtained with unlabeled controls where everything mimics the experimental conditions except that unlabeled substrates are used. By contrasting heavy gradient fractions from isotopically labeled samples relative to corresponding fractions from controls, the identities of microbes with labeled nucleic acids can be
determined 

We used an RNA-Seq differential expression statistical framework
\citep{Love_2014} to find OTUs enriched in heavy fractions of labeled
gradients relative to corresponding density fractions in control gradients
(for review of RNA-Seq differential expression statistics applied to
microbiome OTU count data see \citet{24699258}). We use the term
“differential abundance” (coined by \citet{24699258}) to denote OTUs that
have different proportion means across sample classes (in this case the only
sample class is labeled:control).  CsCl gradient fractions were categorized
as "heavy" or "light". The heavy category denotes fractions with density
values above 1.7125 and below 1.755 g/mL. Since we are only interested in enriched OTUs
(labeled versus control), we used a one-sided wald-test for differential
abundance (the null hypothesis is the labeled:control proportion mean ratio
for an OTU is less than a selected threshold). P-values were corrected with
the Benjamini and Hochberg method \citep{citeulike:1042553}. We selected a
log$_{2}$ fold change null threshold of 0.75 (or a labeled:control proportion
mean ratio of 1.68). DESeq2 was used to calculate the moderated log$_{2}$
fold change of labeled:control proportion mean ratios and corresponding
standard errors for the Wald test. Mean ratio moderation allows for reliable ratio ranking such
that high variance and likely statistically insignificant mean ratios are
appropriately shrunk and subsequently ranked lower than they would be as raw
ratios. Those OTUs that exhibit a statistically significant increase in
proportion in heavy fractions from $^{13}$C-labeled samples relative to
corresponding controls have increased significantly in bouyant density in
response to $^{13}$ treatment.

\subsubsection{Community and Sequence Analysis}
Principal coordinate ordinations depict the relationships between
samples. Weighted Unifrac distances were
used as the sample distance metric for ordination. The Phyloseq
\citep{24699258} wrapper for Vegan \citep{vegan} (both R packages) was
used to compute sample values along principal coordinate axes. GGplot2
\citep{ggplot2} was used to display sample points along the first and
second principal axes. Adonis tests \cite{Anderson_2001} were done with default
number of permutations (1000).

Code to take raw sequencing data through the presented figures (including
download and processing of literature environmental datasets) can be
found at:

\url{http://nbviewer.ipython.org/github/chuckpr/CSIP_succession_data_analysis}